src/menu/views/load_disk.c

```
@@ -162,14 +221,41 @@ void view_load_disk_init (menu_t *menu) {
+        if(menu->load.load_history != -1) {
+            id = menu->load.load_history;
+            items = menu->bookkeeping.history_items;
+        } else if (menu->load.load_favorite != -1) {
+            id = menu->load.load_favorite;
+            items = menu->bookkeeping.favorite_items;
+        }
+
+        menu->load.load_history = -1;
+        menu->load.load_favorite = -1;
+
+        menu->load.disk_path = path_clone(items[id].primary_path);
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(legacyUninitvar)